parity-util-mem/Cargo.toml

```diff
@@ -25,7 +25,7 @@ impl-trait-for-tuples = "0.1.3"
 
 smallvec = { version = "1.0.0", optional = true }
 ethereum-types = { version = "0.8.0", optional = true, path = "../ethereum-types" }
-parking_lot = { version = "0.9.0", optional = true }
+parking_lot = { version = "0.10.0", optional = true }
```

Comment:
Isn't that a breaking change for `parity-util-mem`?

Reply:
Seems like [no](https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api).

Reply:
But that's public API. `parity-util-mem::MallocSizeOf` v0.4.2 is implemented for parking_lot v0.9.0, but `parity-util-mem::MallocSizeOf` v0.4.3 is not (as 0.10.0 is a breaking change to 0.9.0)

Reply:
hmm technically yes, but it will require major bump just for everything
ok to me though

Reply:
yeah, that's version bump creep is unfortunate, but we already have to do this e.g. for `rand` and other libs, so it's better to safe than sorry